modules/mysql/metadata.yaml

```diff
@@ -100,6 +100,11 @@ spec:
       start_time: null
       transaction_log_retention_days: null
     required: false
+  - name: connector_enforcement
+    description: Force the client to use the connector. See [more details](https://cloud.google.com/sql/docs/mysql/connect-connectors)
+    type: bool
+    default: true
```

Comment:
Let's change the default value as false, this is what we want as per documentation and variable file.

Reply:
nice catch.  Updated